realsense2_camera/include/realsense2_camera/constants.h

```diff
@@ -94,6 +94,7 @@ namespace realsense2_camera
     const std::string DEFAULT_ALIGNED_DEPTH_TO_FISHEYE_FRAME_ID = "camera_aligned_depth_to_fisheye_frame";
 
     const bool DEFAULT_USE_FIX_SET_EXPOSURE = true;
```

Comment:
In the launch files you changed used_fix_set_exposure to default to false, should that be reflected here too?

Reply:
Yes this value should be false. Thanks for catching that.